yuhow/Makefile

```diff
@@ -1,76 +0,0 @@
-SHELL := bash
```

Comment:
If there's any issue you find in the Makefile we can work it out together.  The file itself isn't particularly complex.  I hope we can make good use of the automation.